### ISSUE TITLE
Is there a way to convert back a potree dataset into a .LAS ?

### ISSUE THREAD

**Issue by nmenu** [potree/potree#731]
The question is pretty much self explanatory. 

Use case :
I build up large point clouds datasets, most of the time. 
I process the original LAS to put them on a Potree server, so easy access for every one.
Sometimes, but you never no in advance, the Potree access is enough for most use (so I like to delete the original LAS to free up space)... but sometimes, later on (may be long time later after the potree files has been posted to the server, some folks want to use the point cloud in CAD... This is when it would be hand to be able to recreate the LAS from the potree files.

Any suggestions ?
Thanks ++, Nico

**Comment by m-schuetz** [comment]
Dear Nico,

Please always keep the original data, as Potree converts them mainly for visualization purposes with no attempt to faithfully recover the original data. In particular, Potree stores RGB in 1 byte instead of 2 bytes each, and it may not consider all attributes that are available in a las file. It's also not possible to recover the order of points. 

As far as I know, [Entwine](https://entwine.io/) makes sure that data can be converted back faithfully. You can use Entwine as a replacement for the PotreeConverter, and still use Potree to display data converted by Entwine. 

/ping @connormanning

**Comment by connormanning** [comment]
Correct, [Entwine](https://entwine.io) maintains a strict lossless definition, and contains everything you'd need to be able to re-create source tiles, or extract data from a bounds query.  You can use the full power of PDAL to run processing steps and to write output to a variety of formats, with the [EPT Reader](https://pdal.io/stages/readers.ept.html) being used to read Entwine's output.

The losslessness guarantees of Entwine are discussed a bit in the conference talk that is linked on the main page of entwine.io.

**Comment by mcu13321** [comment]
I find a class called LASExporter, and it is used in conjunction with height profile. Is it possible to extend the class to export LAS? 
Thanks

**Comment by Liquidmasl** [comment]
oh damn.

navis ivion only supports streaming its potree files from their platform... ayayay

as this issue is super old; is the new potree format lossless by any chance?
